app/src/main/java/com/dealshot/dealshotandroidapp/Plaza.kt

```diff
@@ -0,0 +1,5 @@
+package com.dealshot.dealshotandroidapp
+
```

Comment:
Bad design for plaza post filtering; should be a singleton class with multiple filtering generation methods.